src/familybot/lib/database.py

```
@@ -19,199 +21,244 @@
+    conn = get_db_connection()
+    with _write_lock:
+        yield conn
+
+
+def close_db_connection():
+    """Close the thread-local database connection if it exists."""
+    if hasattr(_local, "conn") and _local.conn is not None:
+        try:
+            _local.conn.close()
+        except sqlite3.Error:
+            pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(sqlite3.Error)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(sqlite3.Error): ...`

(SIM105)